packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -335,7 +381,34 @@ mod tests {
         // We are expecting a MissingData error because at the moment, the
         // ABIgen expects exactly 4 arguments (see `expand_function_arguments`), here
         // there are 5
-        let _bindings = Abigen::new("test", contract).unwrap().generate().unwrap();
+        let _bindings = Abigen::new("test", contract, None)
+            .unwrap()
+            .generate()
+            .unwrap();
+    }
+
+    #[test]
+    fn generates_deployable() {
```

Comment:
This test does not test anything from what I understand, you should probably add an assert statement or expect a panic from the test

Reply:
The idea was to just have a simple test case which triggers the generation of the `Deployable` implementation for the binding